CONTRIBUTING.md

```diff
@@ -333,6 +333,8 @@ Container system, boot system and library changes are some examples of the pull
 The *Nixpkgs committers* are people who have been given
 permission to merge.
 
+To streamline autoupdates, leverage the nixpkgs-merge-bot by simply commenting `@NixOS/nixpkgs-merge-bot merge` if the package resides in `pkgs/by-name` and the commenter is among the package maintainers. The bot ensures that all ofBorg checks, except for darwin, are successfully completed before merging the pull request. Should the checks still be underway, the bot patiently waits for ofBorg to finish before attempting the merge again.
```

Comment:
Since there are discussions about the future roles of merge bot, I think it should be moire future-proof.

```suggestion
To streamline automated updates, leverage the nixpkgs-merge-bot by simply commenting `@NixOS/nixpkgs-merge-bot merge`. The bot will verify if the following conditions are met, refusing to merge otherwise:
- the commenter that issued the command should be among the package maintainers;
- the package should reside in `pkgs/by-name`.

Further, nixpkgs-merge-bot will ensure all ofBorg checks (except the Darwin-related ones) are successfully completed before merging the pull request. Should the checks still be underway, the bot patiently waits for ofBorg to finish before attempting the merge again.
```